pymc3/tests/test_plots.py

```diff
@@ -29,6 +29,14 @@ def test_plots():
         forestplot(trace)
         plot_posterior(trace)
         autocorrplot(trace)
+        energyplot(trace)
+
+
+def test_energyplot():
+    with asmod.build_model() as model:
+        trace = sample()
+
+        energyplot(trace)
```

Comment:
can you unindent the `energyplot` here?  if it has its own function, maybe you could work out the API a bit, and pass some keywords? testing plots is hard...

Reply:
(and actually in `test_plots` as well?  bad habit to not leave a context manager when you're done with it)

Reply:
I will try to improve this, if I fail you can continue with this :-)